Wikipedia/Code/Session.swift

```diff
@@ -647,16 +649,31 @@ class SessionDelegate: NSObject, URLSessionDelegate, URLSessionDataDelegate {
     
     func urlSession(_ session: URLSession, dataTask: URLSessionDataTask, didReceive response: URLResponse, completionHandler: @escaping (URLSession.ResponseDisposition) -> Void) {
         
-        if let httpResponse = response as? HTTPURLResponse,
-            httpResponse.statusCode == 304 {
+        if let httpResponse = response as? HTTPURLResponse {
```

Comment:
Is this the relevant chunk for resolving the banner issue you were mentioning in task sync? If so, is there a related phab ticket (or original repro steps) to help test the change? Apologies, but I didn't see it in the description linked tickets.

Reply:
Sorry about that! Yes, this is the area of the fix. I added a phab task here - https://phabricator.wikimedia.org/T251566.